src/foremast/pipeline/construct_pipeline_block.py

```diff
@@ -70,13 +70,38 @@ def check_provider_healthcheck(settings, default_provider='Discovery'):
     return ProviderHealthCheck(providers=health_check_providers, has_healthcheck=has_healthcheck)
 
 
+def get_template_name(env, pipeline_type):
+    """Generates the correct template name based on pipeline type
+
+    Args:
+        env (str): environment to generate templates for
+        pipeline_type (str): Type of pipeline like ec2 or lambda
+    
+    Returns:
+        str: Name of template
+    """
+    if pipeline_type == 'ec2':
+        if env.startswith('prod'):
+            template_name = 'pipeline/pipeline_{}.json.j2'.format(env)
```

Comment:
@e4r7hbug ok, good idea. I updated this and collapsed the conditional into just one level.